internal/manager/setup.go

```diff
@@ -275,5 +276,14 @@ func getKongClients(ctx context.Context, cfg *Config) ([]adminapi.Client, error)
 		clients = append(clients, client)
 	}
 
+	if cfg.Konnect.ConfigSynchronizationEnabled {
+		konnectClient, err := adminapi.NewKongClientForKonnectRuntimeGroup(ctx, cfg.Konnect)
+		if err != nil {
+			logger.Error(err, "failed creating Konnect Runtime Group Admin API client, skipping synchronisation")
+		} else {
+			clients = append(clients, konnectClient)
+		}
+	}
```

Comment:
I'm just now realizing that we're adding konnect clients to the same structs/loops as "regular" admin API clients which is OK I guess currently but perhaps we could think about isolating them and running separate synchronization loops for both not clumping them up together which might hinder observability (for this stage through logs but perhaps they could also be distinguished in metrics through labels at some point).

Reply:
Yup, I hope that your PR will get merged sooner than mine and I will be able to adapt to the changes we've got for SCD. I'm not sure if that will require a separate task to run them though, let's see.